runtime/commonTest/src/kotlinx/serialization/json/JsonTestBase.kt

```diff
@@ -135,8 +135,7 @@ abstract class JsonTestBase {
         serializer: KSerializer<T>,
         data: T,
         expected: String,
-        json: Json = unquoted,
-        printResult: Boolean = false
```

Comment:
Why you've deleted this param?

Reply:
`printResult` is neither used nor should be part of the actual code.
It is okay to debug with `println`, but it's better to keep it local